pkg/dbutils/dbutils.go

```diff
@@ -33,6 +35,10 @@ func (o *Oid) Scan(src interface{}) error {
 	return nil
 }
 
+func (o Oid) Value() (driver.Value, error) {
```

Comment:
Hm... do we need those for the query in he FetchRelationInfo to accept dbuils.Oid? Why is it converted to int64 (oids are 4-bytes type, aka uint32)?

Reply:
currently it's broken: `FetchRelationInfo` returns `could not fetch table info: could not exec query: cannot convert 635788 to pguint32`

Reply:
why do we convert it to int64 and not to uint32?

Reply:
pgx expects int64 from Value method of the numeric types,
have a look at oid, int* types in pgx/pgtype